drivers/sensor/lis2du12/lis2du12.h

```diff
@@ -0,0 +1,102 @@
+/* ST Microelectronics LIS2DU12 3-axis accelerometer sensor driver
+ *
+ * Copyright (c) 2023 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Datasheet:
+ * https://www.st.com/resource/en/datasheet/lis2du12.pdf
+ */
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_LIS2DU12_LIS2DU12_H_
+#define ZEPHYR_DRIVERS_SENSOR_LIS2DU12_LIS2DU12_H_
+
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/types.h>
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/sys/util.h>
+#include <stmemsc.h>
+#include "lis2du12_reg.h"
+
+#if DT_ANY_INST_ON_BUS_STATUS_OKAY(spi)
+#include <zephyr/drivers/spi.h>
+#endif /* DT_ANY_INST_ON_BUS_STATUS_OKAY(spi) */
+
+#if DT_ANY_INST_ON_BUS_STATUS_OKAY(i2c)
+#include <zephyr/drivers/i2c.h>
+#endif /* DT_ANY_INST_ON_BUS_STATUS_OKAY(i2c) */
+
+#define LIS2DU12_EN_BIT					0x01
+#define LIS2DU12_DIS_BIT					0x00
```

Comment:
the indentation looks funny, I'd sugget to just use a space

```c
#define LIS2DU12_EN_BIT	 0x01
#define LIS2DU12_DIS_BIT 0x00
```

no need to make it fancy, same below

Reply:
mmmh, I think that maybe tabs are better. But I'd agree to shorten it a bit and align with below

Reply:
Or just a space... it's just to, I wouldn't bother.